src/SeqViz.tsx

```diff
@@ -224,20 +224,6 @@ export default class SeqViz extends React.Component<SeqVizProps, SeqVizState> {
    * If an accession was provided, query it here.
    */
   componentDidMount(): void {
-    if (typeof window !== "undefined") {
-      // Allow the user to choose whether to load fonts from Google Fonts or from another source
-      if (!this.props.disableExternalFonts) {
-        // Fetch Roboto Mono, the only font used by SeqViz (at the time of writing)
-        // https://github.com/typekit/webfontloader/issues/383#issuecomment-389627920
-        /* eslint-disable */
-        require("webfontloader").load({
-          google: {
-            families: ["Roboto Mono:300,400,500"],
-          },
-        });
```

Comment:
This package hasn't been updated in 8 years and apparently doesn't work with ESM updates so I'm removing it completely.  The only thing I can't figure out is how to properly set the font weight of the nucleotides so right now it's slightly bolder than before (see https://github.com/AsimovBio/kernel-universal/pull/8638)